python/python/bystro/vcf_utils/simulate_random_vcf.py

```diff
@@ -0,0 +1,65 @@
+"""VCF index and data simulator for testing analysis modules."""
+
+import random
+from io import StringIO
+
+import pandas as pd
+
+HEADER_COLS = ["#CHROM", "POS", "ID", "REF", "ALT", "QUAL", "FILTER", "INFO", "FORMAT"]
```

Comment:
could we move the comment syntax out of HEADER_COLS?

Reply:
I think '#CHROM' is how VCF is usually coded? The preprocess depends on that designation. Or did you mean something else by this?